package.json

```diff
@@ -12,6 +12,7 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
+    "rest-doc": "redocly bundle external@latest --output external.yaml && redocly build-docs ./external.yaml -o static/rest.html && rm ./external.yaml",
```

Comment:
@jovezhong feel free to add params `redocly build-docs ./external.yaml -o static/rest.html` part . Just make sure you are building with `./external.yaml`

Reply:
better to keep the theme (font,color) under static folder

Reply:
--options=rest_theme.json

Reply:
@jovezhong hmm, I'm getting `Unknown argument: options` when running `redocly build-docs ./external.yaml --options=static/rest_theme.json -o static/rest.html`

I cannot find this options in their doc as well https://redocly.com/docs/cli/commands/build-docs/
I'm using
```
> redocly --version
1.0.2
```

Reply:
I am using `redoc-cli build`

Reply:
https://redocly.com/docs/api-reference-docs/configuration/theming/#example-theme you may have to add it to your redocly.yaml

Reply:
`redoc-cli` is deprecated so I updated the theme https://github.com/timeplus-io/docs/pull/87/commits/f3dad891b377626bd414a81953e2f0280c69bfc9
```
   ┌───────────────────── DEPRECATED ─────────────────────┐
   │                                                      │
   │   This package is deprecated.                        │
   │                                                      │
   │   Use `npx @redocly/cli build-docs <api>` instead.   │
   │                                                      │
   └──────────────────────────────────────────────────────┘
```

Later on we can just `yarn rest-doc`